### PR TITLE
dont use modelExperimental as default

### PR DESCRIPTION
**Pull request by katSchmid**
https://propelleraero.atlassian.net/browse/VN-2046

### CODE REVIEW

**Comment by chris-cooper** [comment]
Are you going to do further investigation of our GLTF authoring code to see if we can support the new Cesium behaviour?

**Comment by katSchmid** [comment]
Up to you?

**Comment by katSchmid** [comment]
Up to you? Even if it works it can change anytime
https://cesium.com/learn/cesiumjs/ref-doc/ModelExperimental.html?classFilter=ModelExperimental

Experimental
This feature is using part of the 3D Tiles spec that is not final and is subject to change without Cesium's standard deprecation policy.

File: Source/Scene/Cesium3DTileContentFactory.js [hunk 1]
```diff
@@ -118,20 +119,28 @@ const Cesium3DTileContentFactory = {
...
     const byteLength = dataView.getUint32(8, true);
     const glb = new Uint8Array(arrayBuffer, byteOffset, byteLength);
-    return ModelExperimental3DTileContent.fromGltf(
-      tileset,
-      tile,
-      resource,
-      glb
-    );
+    //// PROPELLER HACK  rolled back
+    if (tileset.enableModelExperimental) {
```
Comment (chris-cooper) [inline Source/Scene/Cesium3DTileContentFactory.js:123]: Is `tileset` the parsed JSON?  If so, `extras` might be a better place to do this.  https://github.com/CesiumGS/3d-tiles/tree/main/specification#extras

image

**Comment by 3a1b2c3** [comment]
lets see if experimental is still regarded experimental  https://community.cesium.com/t/is-modelexperimental-now-offical-v1-92/17685. Above is a straight rollback to the previous version

File: Source/Scene/Cesium3DTileContentFactory.js (hunk 1 above)
Comment (chris-cooper) [reply Source/Scene/Cesium3DTileContentFactory.js:123]: @katSchmid has pointed out that this is straight from the previous Cesium version https://github.com/CesiumGS/cesium/blob/1.90/Source/Scene/Cesium3DTileContentFactory.js#L20

**Review by chris-cooper — APPROVED** [review]
Looks good :taco:

**Comment by katSchmid** [comment]
This will be fixed when we add support for Image-Based Lighting to ModelExperimental. We expect that to happen soon (either the 1.92 or 1.93 release). [#9838](https://github.com/CesiumGS/cesium/issues/9838)

**Pull request merged**